File: ControlMedico.back-end/ControlMedico.Modelos/Modelos/Cita.cs
using ControlMedico.Modelos.Util;
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ControlMedico.Modelos.Modelos
{
    public partial class Cita
    {
        [Key]
        public int IdCita { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime FechaCita { get; set; }
        public int IdPaciente { get; set; }
        public EnumTipoCita TipoCita { get; set; }
        public bool Cancelada { get; set; }

        [ForeignKey(nameof(IdPaciente))]
        [InverseProperty(nameof(Paciente.Cita))]
        public virtual Paciente IdPacienteNavigation { get; set; }
    }
}

File: ControlMedico.front-end/ControlMedicoAngular/src/app/_models/cita.ts
import { Paciente } from './paciente';

export class Cita {
    
        public IdCita: number;
        public FechaCita: Date;
        public IdPaciente: number;
        public TipoCita: EnumTipoCita;
        public DesTipoCita: string;
        public Cancelada: boolean;
        public InfoPaciente: Paciente;
    
}